<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 184, 60);</string>
  </property>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>0</y>
     <width>401</width>
     <height>520</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="playerPassword">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>160</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 9pt &quot;Microsoft YaHei UI&quot;;
color: red</string>
   </property>
   <property name="text">
    <string>请输入密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="playerName">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>110</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 9pt &quot;Microsoft YaHei UI&quot;;
color: red
</string>
   </property>
   <property name="text">
    <string>用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="ensurePlayerPassword">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>220</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 9pt &quot;Microsoft YaHei UI&quot;;
color: red</string>
   </property>
   <property name="text">
    <string>请再次确认密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="title2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>60</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Microsoft YaHei UI&quot;;
color: red
</string>
   </property>
   <property name="text">
    <string>欢迎来到燃烧的画</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ensurePassword">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>250</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>190</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="sure">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>290</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 9pt &quot;Microsoft YaHei UI&quot;;
color: rgb(85, 170, 255);
background: rgba(255, 255, 255, 180); /* 半透明白色背景 */
            border: 2px solid white; /* 白色实线边框 */
            border-radius: 10px; /* 圆角边框 */
            padding: 2px 4px;</string>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backToLogin">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>340</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 9pt &quot;Microsoft YaHei UI&quot;;
color: rgb(85, 170, 255);
background: rgba(255, 255, 255, 180); /* 半透明白色背景 */
            border: 2px solid white; /* 白色实线边框 */
            border-radius: 10px; /* 圆角边框 */
            padding: 2px 4px;</string>
   </property>
   <property name="text">
    <string>返回至注册页面</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
